packages/assets-controllers/package.json

```diff
@@ -49,6 +49,7 @@
     "@types/uuid": "^8.3.0",
     "async-mutex": "^0.2.6",
     "ethereumjs-util": "^7.0.10",
+    "fast-equals": "^5.0.1",
```

Comment:
Have you considered using `lodash.isEqual` instead? We already have `lodash` here as a dependency

Reply:
Ah. I thought we were only using that in the extension. Ah well. I've switched to that here: [293acac](https://github.com/MetaMask/core/pull/3663/commits/293acaca5988a6bf46045d2cc0df026403f219cf)